lib/sisimai/bite/email/activehunter.rb

```diff
@@ -0,0 +1,124 @@
+module Sisimai::Bite::Email
+  module Activehunter
+    # Sisimai::Bite::Email::Activehunter parses a bounce email which created
+    # by TransWARE Active!hunter.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Activehunter.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\A"MAILER-DAEMON"/,
+        :subject => %r/FAILURE NOTICE :/,
+      }
+      Re1 = {
+        :begin   => %r/\A  ----- The following addresses had permanent fatal errors -----\z/,
+        :error   => %r/\A  ----- Transcript of session follows -----\z/,
+        :rfc822  => %r|\AContent-type: message/rfc822\z|,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'TransWARE Active!hunter'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Layout/EmptyLineBetweenDefs: Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->